plugins/metrics/metrics_workerpool.go

```diff
@@ -32,7 +32,8 @@ var WorkerPoolMetrics = collector.NewCollection(workerPoolNamespace,
 			}
 
 			if deps.Retainer != nil {
-				collected[retainerLabel] = float64(deps.Retainer.WorkerPool().WorkerCount())
+				collected[retainerLabel] = float64(deps.Retainer.BlockWorkerPool().WorkerCount() +
+					deps.Retainer.CommitmentWorkerPool().WorkerCount())
```

Comment:
We could expose the `Workers` in the retainer and just do the same as for the protocol above.

```
for name, wp := range deps.Retainer.Workers.Pools() {
	collected[name] = float64(wp.WorkerCount())
	collected[name] = float64(wp.WorkerCount())
}
```